### ISSUE TITLE
Sublicense the Linux mark to avoid legal issues in the future

### ISSUE THREAD

**Issue by fpesari** [FrancescoCeruti/linux-show-player#224]
Hello,

first, let me premise that I am not a lawyer. Despite this, I know that trademark law can create some issues even when trademarks are used innocently, as in this case.

The Linux trademark is owned by a nonprofit, the Linux Foundation, which is benevolent enough to offer a way to sublicense the Linux mark ([here](https://www.linuxfoundation.org/request-a-sublicense/)).

Now someone could argue, this program did not register a trademark and is not made with the explicit goal of making money. Well, according to the Linux Foundation, this does not matter ([source](https://www.linuxfoundation.org/the-linux-mark/#toggle-id-1)):
> You need to apply for a sublicense if you are using the term “Linux” as part of your own trademark or brand identifier for Linux-based software goods or services. It doesn’t matter if your trademark is unregistered, or if you do not plan to make any money using the mark.

Someone else could argue, "but the Linux Foundation is a nonprofit, even if linux-show-player is infringing its trademark they will never do anything about a FLOSS project that's not making big money". To that I would answer, why take any risks?

In short, if this program did not have a web domain and were not packaged in some distros, it would be probably better to rename it to avoid these issues altogether. But since it has some traction and the Linux Foundation offers a way to sublicense the Linux mark, I think renaming can be avoided by obtaining a sublicense.

**Comment by FrancescoCeruti** [comment]
Thanks al lot for bringing this to my attention!

I've already submitted a request :-)

**Comment by phedders** [comment]
IANAL either; is LSP a trademark? or just the working name of a software project? LSP does not use or distribute linux source code or binaries so the only relation is the name used and since I dont believe there is a trademark... my understanding is that a sublicence is not required even in the US. For the !US is it not mostly a non-conversation?

**Comment by fnetX** [comment]
@phedders The examples of trademark requirements include examples like "Real Time Linux Consultants" and while they might technically "distribute" linux binaries, I think the trademark applies for every product or services with "Linux" in the name.

To the domain thing and "unless you use your Linux domain name as a trademark": As far as I understand, this is for the case where your trademark is different from your domain. Consider my fictional Linux distribution called "FictionalOS". I might want to use the domain name fictional-linux.invalid to host it and make sure people know what it is, maybe some other good domain was already taken. But since I'm not using "Fictional Linux" as a brand, but rather "FictionalOS", I do not have to apply for a trademark. The only "Linux" in the name is in the domain. And referring Linux on the page as in "Linux Distribution" would probably fall under the Fair use ("Linux® Distribution" and the trademark notice in the footer).

**Comment by FrancescoCeruti** [comment]
I've never update you on the matter.

I've submitted the request two times, never received any kind response, not sure what to think of it.

If anyone have done this before or known someone that could help, let me know :)
